examples/next/getting-started/src/faust.config.js

```diff
@@ -6,10 +6,14 @@ if (!process.env.NEXT_PUBLIC_WORDPRESS_URL) {
   );
 }
 
+function getSecretKey() {
```

Comment:
This function not part of the old Faust so users trying to adopt the `FAUST_SECRET_KEY` will get confused if they follow the docs